features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/publisher.commons.js

```diff
@@ -124,8 +124,35 @@ $(document).ready(function() {
 		$(this).next().click();
 	});
 
-	$('input[type="file"]').change(function() {
-		$(this).closest('.tbl-upload').find('.txt-filepath').val($(this).val());
+    $('input[type="file"][accept!="image/*"]').change(function(evt) {
```

Comment:
Formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files